src/components/portal/portal.tsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react'
+import { createPortal } from 'react-dom'
+
+const mount = document.getElementById('portal-root')!
+const el = document.createElement('div')
+
+export const Portal = ({ children }: React.PropsWithChildren<{}>) => {
+  useEffect(() => {
+    mount.appendChild(el)
```

Comment:
What will happen if there two <Portals /> on the page at same time?
May be we can create portal element on first mount? 
Something like this:
```
  const portalRootRef = useRef<HTMLDivElement>()
  if (!portalRootRef.current) {
    portalRootRef.current = document.createElement('div')
    portalsRoot.appendChild(portalRootRef.current)
  } 
  
  useEffect(() => () => { portalRootRef.current?.remove() })
  
  return createPortal(children, portalRootRef.current)
```

Reply:
Doesn't seem to be working as expected

Reply:
But you don't prevent this behaviour (